tools/Makefile.am.inc

```diff
@@ -30,3 +30,19 @@ eos_network_id_CFLAGS = \
 	$(NULL)
 
 eos_network_id_LDADD = $(EOS_INSTRUMENTATION_LIBS)
+
+bin_SCRIPTS = eos-label-location
+
+do_edit = sed \
+	-e "s|\@sysconfdir\@|$(sysconfdir)|"
+
+eos-label-location: tools/eos-label-location.in Makefile
```

Comment:
Why not put the build output in the `tools/` directory (in `builddir`) as well? It’s a bit confusing for the source and build product to be in different directories.